features/step_definitions/us13_borrowers_personal_info.rb

```diff
@@ -69,6 +69,7 @@
   @deed_hash = {
     title_number: 'DT567568',
     md_ref: 'e-MD123G',
+    identity_checked: 'Y',
     borrowers: [
       {
         forename: 'Katie',
```

Comment:
Well remembered on changing these steps where we are  setting the borrowers to something different to the default. Though there is also an instance of this in the step def file us40_add_md_ref.rb which hasn't been changed. Probably didnt come up as a problem as it is a test that is supposed to get a 400 back, but it should be getting it back for the specific reason for the scenario, in this instance no md-ref has been supplied not because no identity check indicator has been set, if this makes sense?